source/configure/database-configuration-settings.rst

```diff
@@ -376,7 +376,7 @@ Replica lag settings
 | String array input consists of:                        |                                                                                  |
 |                                                        |                                                                                  |
 | - ``DataSource``: The database credentials to connect  |                                                                                  |
-|   to the replica instance.                             |                                                                                  |
+|   to the database instance.                            |                                                                                  |
```

Comment:
This was very intentionally changed. As I dug into each query we provided, all the guidance said that they can be run on any node for AWS Aurora and MySQL Group replication or the primary for Postgres replication slots

Reply:
Aurora - https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora_global_db_instance_status.html

> You can run this function from any instance in an Aurora PostgreSQL DB cluster or an Aurora PostgreSQL global database. The function returns details about lag for all replica instances.

MySQL Group Replication - https://dev.mysql.com/doc/refman/8.0/en/group-replication-replication-group-member-stats.html

> The information is shared between all the server instances that are members of the replication group, so information on all the group members can be queried from any member.